version.json

```diff
@@ -1,56 +1,60 @@
 {
     "description": "The release value identifies the base version that will be applied via the tools/release-tools/setVersion.js",
     "usage": "When creating a new release you should update this value directly or via the eg. 'npm run setVersion -- 3.2.0' or 'npm run setVersion -- -patch' or 'npm run setVersion -- -minor'",
-    "release": "2.8.8",
+    "release": "2.8.9",
     "next": "patch",
     "pkgs": {
         "@microsoft/applicationinsights-web": {
             "package": "package.json",
-            "release": "2.8.8"
+            "release": "2.8.9"
         },
         "@microsoft/applicationinsights-web-basic": {
             "package": "AISKULight/package.json",
-            "release": "2.8.8"
+            "release": "2.8.9"
         },
         "@microsoft/applicationinsights-channel-js": {
             "package": "channels/applicationinsights-channel-js/package.json",
-            "release": "2.8.8"
+            "release": "2.8.9"
         },
         "@microsoft/applicationinsights-analytics-js": {
             "package": "extensions/applicationinsights-analytics-js/package.json",
-            "release": "2.8.8"
+            "release": "2.8.9"
         },
         "@microsoft/applicationinsights-clickanalytics-js": {
             "package": "extensions/applicationinsights-clickanalytics-js/package.json",
-            "release": "2.8.8"
+            "release": "2.8.9"
         },
         "@microsoft/applicationinsights-debugplugin-js": {
             "package": "extensions/applicationinsights-debugplugin-js/package.json",
-            "release": "2.8.8"
+            "release": "2.8.9"
         },
         "@microsoft/applicationinsights-dependencies-js": {
             "package": "extensions/applicationinsights-dependencies-js/package.json",
-            "release": "2.8.8"
+            "release": "2.8.9"
         },
         "@microsoft/applicationinsights-perfmarkmeasure-js": {
             "package": "extensions/applicationinsights-perfmarkmeasure-js/package.json",
-            "release": "2.8.8"
+            "release": "2.8.9"
         },
         "@microsoft/applicationinsights-properties-js": {
             "package": "extensions/applicationinsights-properties-js/package.json",
-            "release": "2.8.8"
+            "release": "2.8.9"
         },
         "@microsoft/applicationinsights-common": {
             "package": "shared/AppInsightsCommon/package.json",
-            "release": "2.8.8"
+            "release": "2.8.9"
         },
         "@microsoft/applicationinsights-core-js": {
             "package": "shared/AppInsightsCore/package.json",
-            "release": "2.8.8"
+            "release": "2.8.9"
         },
         "@microsoft/applicationinsights-chrome-debug-extension": {
             "package": "tools/chrome-debug-extension/package.json",
-            "release": "0.3.8"
+            "release": "0.3.9"
+        },
+        "applicationinsights-web-config": {
```

Comment:
I'll need to update the build / release pipelines to ensure that this gets published.